Cargo.lock

```diff
@@ -120,9 +126,9 @@ checksum = "4b46cbb362ab8752921c97e041f5e366ee6297bd428a31275b9fcf1e380f7299"
 
 [[package]]
 name = "anstream"
-version = "0.6.17"
+version = "0.6.18"
```

Comment:
We only do `cargo update` during releases or in a separate PR, could you revert this file and only keep changes required for your pr, please?=)